core/tee/tee_rpmb_fs.c

```diff
@@ -1099,6 +1099,10 @@ static TEE_Result tee_rpmb_init(uint16_t dev_id)
 				       RPMB_KEY_MAC_SIZE);
 		if (res != TEE_SUCCESS)
 			goto func_exit;
+#ifdef CFG_RPMB_WRITE_KEY
+		else
+			DHEXDUMP(rpmb_ctx->key, RPMB_KEY_MAC_SIZE);
```

Comment:
1. Perhaps add a message before? ("RPMB key value:")
2. I'd rather print the key only when we try to program it, in order to reduce the risk of leaking it.

Reply:
1. Oops, yes!
2. Ok. Moved the hexdump to `tee_rpmb_write_and_verify_key()` instead which is simpler.

> Also, please add to the commit log that the key is printed with severity TRACE_DEBUG (hence a release build will not leak the key).

Ok, will add during squash.